docs/usage/sat_bootprep.md

```diff
@@ -516,20 +516,20 @@ Only one layer is shown for brevity.
 configurations:
 - name: "{{default.note}}compute-{{recipe.version}}{{default.suffix}}"
   layers:
-  - name: cos-compute-{{cos.working_branch}}
-    playbook: cos-compute.yml
+  - name: cne-compute-{{cne.working_branch}}
+    playbook: cne-compute.yml
     product:
-      name: cos
-      version: "{{cos.version}}"
-      branch: "{{cos.working_branch}}"
+      name: cne
```

Comment:
I think this is right, but we should double-check that the CNE product release distribution tar file does provide the Ansible content moving forward. I'm guessing it most likely does since we've already confirmed the product tar file that provides the image recipes is named cne.

Reply:
Per Dean's email, this is correct.